Makefile

```diff
@@ -21,72 +21,69 @@ RUST_ACTION ?= run -p sigstore-protobuf-specs-codegen
 # generate all language protobuf code
 all: go python typescript ruby jsonschema rust
 
+CUSER=$(shell id -u ${USER})
+CGROUP=$(shell id -g ${USER})
+DOCKER_RUN=docker run --platform linux/amd64 -u ${CUSER}:${CGROUP} -v ${PWD}:/defs
```

Comment:
Here and the line below is where the current system user-id is passed @loosebazooka